.prettierignore

```diff
@@ -6,7 +6,7 @@ operators.md
 dataframes.md
 nushell_map.md
 *_bash*.md
-book/commands/*.md
+commands/commands/*.md
```

Comment:
commands/docs/*.md

Reply:
Indeed, otherwise prettier formats the auto-generated files.